src/main/java/rs/hybridit/serviceImpl/LibraryServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package rs.hybridit.serviceImpl;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import rs.hybridit.model.Library;
+import rs.hybridit.repository.LibraryRepository;
+import rs.hybridit.service.LibraryService;
+
+import java.util.List;
+
+@Service
+public class LibraryServiceImpl implements LibraryService {
+    @Autowired
+    LibraryRepository libraryRepository;
+
+    @Override
+    public Library getOne(long id) {
+        return libraryRepository.getOne(id);
+    }
+
+    @Override
+    public List<Library> getAll() {
+        return libraryRepository.findAll();
+    }
+
+    @Override
+    public Library create(Library library) {
+        return libraryRepository.save(library);
+    }
+
+
+    @Override
+    public void delete(Library library) {
+        libraryRepository.delete(library);
+    }
+}
```

Comment:
We should add some basic unit tests, will explain live

Reply:
Read first: https://www.petrikainulainen.net/programming/testing/writing-clean-tests-naming-matters/